pkg/datapath/linux/bigtcp/bigtcp.go

```diff
@@ -137,6 +138,22 @@ func haveIPv6MaxSize() bool {
 	return false
 }
 
+func probeTSOMaxSize() int {
+	maxSize := math.IntMin(bigTCPGSOMaxSize, bigTCPGROMaxSize)
+	for _, device := range option.Config.GetDevices() {
+		link, err := netlink.LinkByName(device)
+		if err == nil {
+			tso := link.Attrs().TSOMaxSize
+			tsoMax := int(tso)
+			if tsoMax > defaultGSOMaxSize && tsoMax < maxSize {
```

Comment:
Could it happen that `link.Attrs().TSOMaxSize` is less than 65536? If yes, shouldn't we set `maxSize` to the lower value? If not, then the `tsoMax > defaultGSOMaxSize` check is not required.

Reply:
If it is less than that, we get a failure later and won't touch it further.